styles/build.js

```diff
@@ -0,0 +1,105 @@
+const StyleDictionary = require('style-dictionary').extend('./sd.config.js');
+
+StyleDictionary.registerFormat({
+  name: 'themable-scss/web',
+  formatter: (tokens) => {
+    return tokens.allProperties
+      .map((token) => {
+        let tokenName = token.name;
+        let tokenValue = token.original.value;
+        const tokenGroup = token.group;
+        const tokenType = token.type;
+
+        if (tokenName === 'sprk-masthead-translate-y')
+          tokenName = 'sprk-masthead-translateY';
```

Comment:
y did we have to do this?

Reply:
@bobwise, we have that one variable that has a weird naming and thus gets translated incorrectly. I'll add a comment above it in the code explaining.